src/Domain/Exception/DuplicateEntityException.php

```diff
@@ -9,8 +9,13 @@
  */
 final class DuplicateEntityException extends \RuntimeException implements DomainExceptionInterface
 {
-    public static function createForId(string $entity, $id): self
+    public static function createForId(string $entity, $id, ...$idN): self
```

Comment:
Isn't it easier to just change the signature to `createForId(string $entity, ...$idN)`?